src/phy/sys/bpf.rs

```diff
@@ -0,0 +1,129 @@
+use std::io;
+use std::os::unix::io::{AsRawFd, RawFd};
+
+use libc;
+
+use super::{ifreq, ifreq_for};
+
+/// set interface
+const BIOCSETIF: libc::c_ulong = 0x8020426c;
```

Comment:
Surely these hardcoded ioctl values are not valid for every Unix OS, are they?

Reply:
I could not find out their values for other OSes, the most I can do is change this to only compile on macOS if that is what you are implying.